Content.Shared/Consent/ConsentTogglePrototype.cs

```diff
@@ -1,13 +1,38 @@
 namespace Content.Shared.Consent;
 
+using System;
 using Robust.Shared.Prototypes;
 
 /// <summary>
 /// TODO
 /// </summary>
 [Prototype("consentToggle")]
-public sealed partial class ConsentTogglePrototype : IPrototype
+public sealed partial class ConsentTogglePrototype : IPrototype, IComparable
 {
     [IdDataField]
     public string ID { get; private set; } = default!;
+
+    [DataField("category")]
+    public string Category { get; private set; } = default!;
+
+    [DataField("priority")]
+    public int priority { get; private set; } = 0;
+
+    public int CompareTo(object? obj) { // Allow for granular sorting to make the menu display consistently and intuitively
```

Comment:
Missing override modifier:
```suggestion
    public override int CompareTo(object? obj) { // Allow for granular sorting to make the menu display consistently and intuitively
```

Reply:
This is actually giving me an error when compiling, saying "no suitable method found to override". Could it be that the signature is incorrect and a different signature should be used? Or is it more likely that, since I'm pulling the IComparable interface in for this prototype specifically, and prototypes normally aren't comparable like this, that there isn't anything for it to override?

Reply:
Uh, I might be wrong and C# might not actually support the override modifier for virtual methods coming from an interface. If that's the case, this suggestion is not relevant.

Reply:
Fair enough! I appreciate it being brought up, I'd rather have it intentionally not there then just forgotten or assumed.